gtsam/sfm/mfas.h

```diff
@@ -0,0 +1,65 @@
+/*
+  This file defines functions used to solve a Minimum feedback arc set (MFAS)
```

Comment:
Please update the file brief to the GTRC copyright. Do keep the comment about how this is taken from another source.